libs/@local/hash-graph-types/graph_types/_schema.py

```diff
@@ -91,23 +105,16 @@ async def create_model(
         *,
         actor_id: UUID,
         graph: "GraphAPIProtocol",
-    ) -> Annotated[Any, ...]:
-        ty = await self.items.create_model(actor_id=actor_id, graph=graph)
+    ) -> type[list[T]]:
```

Comment:
Reminder for myself. This annotation is incorrect

Reply:
What would be the correct annotation?

Reply:
`type[list[BaseModel]]` as we don't know what the call `create_model` will return